tests/acceptance/standard/test_hooks.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+from typing import TYPE_CHECKING
+
+from tests.acceptance import run_gitlabform, get_random_name
+
+
+@pytest.fixture(scope="class")
+def urls():
+    first_name = get_random_name("hook")
+    second_name = get_random_name("hook")
+    first_url = f"http://hooks/{first_name}.org"
+    second_url = f"http://hooks/{second_name}.org"
+    return first_url, second_url
+
+
+class TestHooksProcessor:
+    def get_hook_from_url(self, project, url):
+        return next(h for h in project.hooks.list() if h.url == url)
+
+    def test_hooks_create(self, gl, project, urls):
+        target = project.path_with_namespace
+        first_url, second_url = urls
+
+        test_yaml = f"""
+            projects_and_groups:
+              {target}:
+                hooks:
+                  {first_url}:
+                    push_events: false
+                    merge_requests_events: false
+                  {second_url}:
+                    job_events: false
+                    note_events: false
```

Comment:
I believe default in GitLab when setting up hooks is false. Might make sense to set them to true. Or, can be mix of both.

Reply:
Push events defaults to true, but yeah, all the other to false...